sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -460,8 +460,8 @@ public String getEntityPath() {
      * @return A peeked {@link ServiceBusReceivedMessage}.
      * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/message-browsing">Message browsing</a>
      */
-    public Mono<ServiceBusReceivedMessage> browse() {
-        return browse(receiverOptions.getSessionId());
+    public Mono<ServiceBusReceivedMessage> peek() {
```

Comment:
Oh, why are we changing this back to `peek`?

Reply:
Yes. we're committed now. :)